SmartSaver.Domain/Repositories/Transaction/ITransactionRepo.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SmartSaver.EntityFrameworkCore.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SmartSaver.Domain.Repositories
+{
+    public enum CreateTransactionResponse
+    {
+        Success,
+        BadAmountError,
+        GeneralError
+    }
+
+    public interface ITransactionRepo
+    {
+        /// <summary>
+        /// Method that returns all account transactions from spending category.
+        /// </summary>
+        /// <param name="accId"></param>
+        /// <returns>Spendings</returns>
+        List<TransactionDto> GetThisMonthAccountSpendings(int accId);
```

Comment:
I would make method for gettting account's transactions. And then fill the TransactionCounterService class with this type of methods